tasks/runtime.yml

```diff
@@ -132,6 +132,17 @@
   register: ohpc_gres_conf
   # NB uses restart rather than reload as this is needed in some cases
 
+- name: Create cgroup.conf
+  # appears to be required even with NO cgroup plugins: https://slurm.schedmd.com/cgroups.html#cgroup_design
+  copy:
+    remote_src: yes
+    src: /etc/slurm/cgroup.conf.example
```

Comment:
Should we ship a template for this now in readiness for supporting it in the future?

Reply:
I think here we just adopt the example one, which seems OK to keep Slurm happy for now. This might make build your own slurm harder I guess.